time    tick      # Time
float64 gyr_x     # Body-frame X angular velocity (rads/sec)
float64 gyr_y     # Body-frame Y angular velocity (rads/sec)
float64 gyr_z     # Body-frame Z angular velocity (rads/sec)
float64 acc_x     # Body-frame X linear acceleration (m/sec^2)
float64 acc_y     # Body-frame Y linear acceleration (m/sec^2)
float64 acc_z     # Body-frame Z linear acceleration (m/sec^2)
